test/cli/rbi-gen-single/family/bart/__package.rb

```diff
@@ -3,8 +3,11 @@
 class Family::Bart < PackageSpec
 
   import Family
+  import Family::Bart::Slingshot
   import Util
+  test_import Util::Testing
 
   export Family::Bart::Character
+  export Test::Family::Bart::BartTest
```

Comment:
This is important for checking that we handle test namespaces properly, but such a line would have no actual effect on typechecking, right?  (Just double-checking my own understanding here.)

Reply:
It would for packages that depend on that symbol from the `bart` package in their test package.